scripts/docker_cmake

```diff
@@ -24,7 +24,7 @@ ARGS=(
     -D TPL_ENABLE_Libmesh=ON
         -D Libmesh_INCLUDE_DIRS=$LIBMESH_DIR/include
         -D Libmesh_LIBRARY_DIRS=$LIBMESH_DIR/lib
-    -D TPL_ENABLE_MOAB=ON
+    -D TPL_ENABLE_MOAB=${ENABLE_MOAB}
```

Comment:
don't really like that we have to do that...  could't we just pass extra flags `-D DataTransferKit_ENABLE_MOABAdapters=OFF` or so?